x-pack/plugins/osquery/cypress/e2e/all/alerts.cy.ts

```diff
@@ -272,16 +269,52 @@ describe('Alert Event Details', () => {
     cy.visit('/app/osquery');
     closeModalIfVisible();
   });
-  // TODO think on how to get these actions triggered faster (because now they are not triggered during the test).
-  // it.skip('sees osquery results from last action', () => {
-  //   cy.visit('/app/security/alerts');
-  //   cy.getBySel('header-page-title').contains('Alerts').should('exist');
-  //   cy.getBySel('expand-event').first().click({ force: true });
-  //   cy.contains('Osquery Results').click();
-  //   cy.getBySel('osquery-results').should('exist');
-  //   cy.contains('select * from uptime');
-  //   cy.getBySel('osqueryResultsTable').within(() => {
-  //     checkResults();
-  //   });
-  // });
+
+  it('should substitute parameters in investigation guide', () => {
+    cy.visit('/app/security/alerts');
+    cy.getBySel('header-page-title').contains('Alerts').should('exist');
+    cy.getBySel('expand-event')
```

Comment:
I see this was copied over, and you have moved some piece of code to a helper before, maybe this piece could also go into a helper? What do you think ? 
I mean the part of waiting for alerts + clicking the first item.

Reply:
I'm not sure about the name, any ideas? :)